basic_pipeline/2_SystemArchitecture.md

```diff
@@ -0,0 +1,25 @@
+Once we know what should be done, let's start thinking how should our system look like!
```

Comment:
I would consider restructuring this chapter. In fact, we could actually split it into two chapters. The first one would be an introduction to membrane elements, distinguishing between different categories (source, filter, sink) and types - depayloader, ordering buffer, mixer. Then in the second chapter we could introduce the system architecture and explain more precisely what each element will be responsible for in this pipeline.

Reply:
Hmm, I wonder if isn't it easier to describe how depayloader and the work basing on our example pipeline?